[2022-03-15 14:09:58,460] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [queued]>
[2022-03-15 14:09:58,472] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [queued]>
[2022-03-15 14:09:58,472] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 14:09:58,472] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 14:09:58,472] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 14:09:58,482] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 14:10:00+00:00
[2022-03-15 14:09:58,486] {standard_task_runner.py:52} INFO - Started process 26072 to run task
[2022-03-15 14:09:58,493] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T14:10:00+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpwn9qxr_z', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmp5vs17yh_']
[2022-03-15 14:09:58,496] {standard_task_runner.py:77} INFO - Job 324: Subtask test
[2022-03-15 14:09:58,554] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [running]> on host MacBook-Pro.local
[2022-03-15 14:09:58,605] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T14:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T14:10:00+00:00
[2022-03-15 14:09:58,607] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 14:09:58,609] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 14:09:58,618] {subprocess.py:85} INFO - Output:
[2022-03-15 14:09:58,623] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 14:09:58,625] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 14:09:58,650] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T141000, start_date=20220315T190958, end_date=20220315T190958
[2022-03-15 14:09:58,680] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 14:09:58,713] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-15 14:15:03,841] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [queued]>
[2022-03-15 14:15:03,849] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [queued]>
[2022-03-15 14:15:03,849] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 14:15:03,849] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 14:15:03,849] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 14:15:03,859] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 14:10:00+00:00
[2022-03-15 14:15:03,864] {standard_task_runner.py:52} INFO - Started process 26178 to run task
[2022-03-15 14:15:03,870] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T14:10:00+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmps5bx8hfz', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpjq4k3e4s']
[2022-03-15 14:15:03,873] {standard_task_runner.py:77} INFO - Job 324: Subtask test
[2022-03-15 14:15:03,926] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T14:10:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-15 14:15:03,981] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T14:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T14:10:00+00:00
[2022-03-15 14:15:03,983] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 14:15:03,984] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 14:15:03,995] {subprocess.py:85} INFO - Output:
[2022-03-15 14:15:04,003] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 14:15:04,005] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 14:15:04,027] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T141000, start_date=20220315T191503, end_date=20220315T191504
[2022-03-15 14:15:04,051] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 14:15:04,075] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
